src/main/utils/PlaybackAPI.js

```diff
@@ -134,6 +138,17 @@ class PlaybackAPI {
     this._save();
   }
 
+  _setPauseAfter(mode) {
+    if (mode) {
```

Comment:
The parameter "mode" is very non descript.  I have no idea what it is by looking at it.  You may have copied from repeat or shuffle functions but for those a "repeat mode" and a "shuffle mode" is actually a thing.